instat/dlgReplaceValues.vb

```diff
@@ -236,9 +236,16 @@ Public Class dlgReplaceValues
 
     End Sub
 
+    Private Sub CheckedRadio()
+        If (rdoNewFromBelow.Checked AndAlso rdoNewFromBelow.Enabled = False) OrElse (rdoNewFromAbove.Checked AndAlso rdoNewFromAbove.Enabled = False) Then
+            rdoNewMissing.Checked = True
+        End If
+    End Sub

```

Comment:
We were having an issue that a disabled rdo could be checked. This sub helps solve that problem. I thought to set it to NA in this case because that's the default. Does that sound reasonable to you?